client/src/components/Header/Header.js

```diff
@@ -21,6 +24,11 @@ function Header() {
         <div className='searchBar'>
           <input
             onChange={e => setSearchTerm(e.target.value)}
+            onKeyPress={event => {
```

Comment:
insteaf of 
```javascript
onKeyPress={event => {
                if (event.key === 'Enter') {
                  window.location = handleLink()
                }
              }}
```

change to 
```javascript
onKeyPress={event => event.key === 'Enter' && window.location = handleLink()
```

Reply:
Parsing error: Invalid left-hand side in assignment expression
this error

Reply:
ahh this is a linter problem, yeah for now don't make the change